### ISSUE TITLE
 Add Zoom In and Zoom Out Feature to Solana Playground

### ISSUE THREAD

**Issue by hardiktheprogrammer** [solana-playground/solana-playground#122]
### This feature could be accessed through a header or Menu Bar

**Comment by hardiktheprogrammer** [comment]
yeah right but while the browser's native zoom settings can be useful for zooming in and out of web pages, they may not always provide the optimal user experience, especially when it comes to zooming in on specific sections of a page or when the page's font size is too small. In addition, some users may not be aware of or may have difficulty using the keyboard shortcuts or mouse gestures required to zoom in and out.

**Comment by hardiktheprogrammer** [comment]
@acheroncrypto  so Can I work on Zoom In and Zoom out feature

**Comment by acheroncrypto** [comment]
Sure, go ahead! Are you planning to make specific sections zoomable?

**Comment by hardiktheprogrammer** [comment]
yeah only on  the code editor section of the IDE because you can zoom Code Editor and give better User experience
